man/fit.cosh.Rd

```diff
@@ -0,0 +1,96 @@
+\name{fit.cosh}
```

Comment:
Why are you fitting a sum of cosh to an effective mass?

Reply:
I fit the correlators, but I use the effMass class to do so. This is because it is an alternative for fit.effectivemass and can be used in precisely the same way. The idea is that the fit result can be compared to the effective mass plateaus directly.

Reply:
Okay, but then this should also support fitting to a bootstrapped ```cf``` directly.